lib/transrate/assembly.rb

```diff
@@ -59,6 +59,10 @@ def initialize file
           logger.error ">#{contig.name}"
           logger.error "Please make sure there are no duplicate entries in the assembly"
           logger.error "Contig name is taken from before the first | or space"
+          logger.error "If you used Trinity, there is a known bug that breaks" +
+                       "contig names to make them non-unique."
+          logger.error "You can fix your Trinity assembly by replacing | with _"
+          logger.error "e.g. `sed 's/\\|/_/' Trinity.fa > Trinity.fixed.fa`"
```

Comment:
Escaping the pipe works on OSX but messes up the formatting on linux (for some reason the underline is put at the beginning of each line).

`sed 's/|/_/'` seems to work as expected for both!